### PR TITLE
fix(sbb-toggle-check): scroll into view when focused via tab key

### LINKED ISSUE #1639: [Bug] (sbb-dialog): scroll behavior with tab
currently the sbb-dialog doesn't scroll when we navigate with tab

### PR DESCRIPTION
**Pull request by dauriamarco**
Closes #1639

### CODE REVIEW

**Review by jeripeierSBB — APPROVED** [review]
LGTM, but should we also check other components?

File: src/components/sbb-toggle-check/sbb-toggle-check.tsx [hunk 1]
```diff
@@ -87,19 +87,19 @@ export class SbbToggleCheck implements ComponentInterface, AccessibilityProperti
...
-          required={this.required}
-          checked={this.checked}
-          value={this.value}
-          onChange={(event: Event): void => this.checkedChanged(event)}
-          aria-label={this.accessibilityLabel}
-        />
-        <label class="sbb-toggle-check" htmlFor="sbb-toggle-check-input">
+        <label class="sbb-toggle-check">
+          <input
+            id="sbb-toggle-check-input"
```
Comment (jeripeierSBB) [inline src/components/sbb-toggle-check/sbb-toggle-check.tsx:92]: Do we still need the id?

**Comment by dauriamarco** [comment]
> LGTM, but should we also check other components?

I've checked and I think the only other case where we don't have the input nested in the label is with form fields, but I've tested them and they seem to already work correctly.

File: src/components/sbb-toggle-check/sbb-toggle-check.tsx (hunk 1 above)
Comment (dauriamarco) [reply src/components/sbb-toggle-check/sbb-toggle-check.tsx:92]: Probably not 👍

**Review by kyubisation — APPROVED** [review]
LGTM 👍

File: src/components/sbb-toggle-check/sbb-toggle-check.e2e.ts [hunk 2]
```diff
@@ -46,9 +46,6 @@ describe('sbb-toggle-check', () => {
 
       // Although the inner native button receives the focus, the active element is the host
       expect(await page.evaluate(() => document.activeElement.id)).toBe('outer-id');
-      expect(await page.evaluate(() => document.activeElement.shadowRoot.activeElement.id)).toBe(
```
Comment (jeripeierSBB) [inline src/components/sbb-toggle-check/sbb-toggle-check.e2e.ts:49]: maybe check for tagName here?

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**